.github/README.md

```diff
@@ -14,44 +14,41 @@ The requirements are intentionally sparse (the devil is in the detail). We don't
 
 ## The Rules
 
-- Read and understand [The Challenge](#the-challenge)
-- Create a solution that satisfies the [Requirements](#requirements)
-- Send the link to the repository containing your solution to careers@q-ctrl.com
+- Read and understand [The Challenge](#the-challenge).
+- Create a solution that satisfies the [Requirements](#requirements).
+- Send the link to the repository containing your solution to the person who contacted you.
```

Comment:
Good idea - but we have to specify who they need to allow access to. Is there a way to create some sort of public group they can allow access to?

Reply:
I don't think there's an 'easy' way without just directly sharing access with an email/github user

Reply:
If we want to prevent that we could just instruct them to not mention Q-CTRL on the repo name, description or code, for instance. Not sure how realistic this request would be, but sharing private repos can get messy and would be harder to share internally for others to contribute to the review.

Reply:
I'll see if there is some easy way to share with a group, otherwise I think the above comment is probably the way to go.